README.md

```diff
@@ -1,5 +1,8 @@
-# js-ipfs-merkle-dag
+# [DEPRECATED] js-ipfs-merkle-dag
 
+**DEPRECATED** Please use https://github.com/ipld/js-ipld-dag-pb/ instead
```

Comment:
use `#` instead to make it very clear

Also point to `IPLD Resolver`, because this module was both format + dag service.